backend/blueprint.py

```diff
@@ -154,6 +154,11 @@ def get_import_list(info_role):
             else:
                 date_end_import = r.date_end_import
 
+            auteurStr = r.auteur[0].nom_role + ' ' + r.auteur[0].prenom_role
```

Comment:
L'élément 0 du tableau est traité deux fois ici. A la ligne 157 puis dans la boucle.
Pour séparer les élément d'un tableau par un ";" tu peux faire comme ça
```suggestion
           auteurStr = ' ; '.join([r.nom_role + '  ' + r.prenom_role for a in r.auteur ])
```
Est-ce qu'il est possible de passer le noms de la colonne (relationships) en anglais (author)

Reply:
Oups. Oui je vais faire ça